first-between-santa_clara-san_fernando.html

```diff
@@ -9,39 +9,40 @@
   <body>
     <h1>You are on First Street between Santa Clara and San Fernando Street</h1>
     <p>
-      While walking down first street you see many businesses and tram heading northbound on a track. You see Good
-      karma, a vegan friendly eatery with great craft beers to your west. On the
-      west side of the street there is also a subway, an ice cream sandwich shop
-      called CREAM, then a street that branches off about halfway down the
-      block. south of the street branching off to the west theres Fifty-Five South Bar.
-      Peering inside of fifty-five south you see many unique liqours.
+      While walking down first street you see many businesses and tram heading
+      northbound on a track. You see Good karma, a vegan friendly eatery with
+      great craft beers to your west. On the west side of the street there is
+      also a subway, an ice cream sandwich shop called CREAM, then a street that
+      branches off about halfway down the block. south of the street branching
+      off to the west theres Fifty-Five South Bar. Peering inside of fifty-five
+      south you see many unique liqours.
     </p>
     <p>
       On the east side of the street you see an alleyway going east, a cookie
       bar called Insomnia, a restaurant called Nomkai eatery, and then a few
       bars in succession. The bars you see are The Temple bar and lounge which
       seems upon first glace like a upscale club, the Fuze lounge which is very
-      lit up and musical, Still OG and Alter Ego which looks like a hole in the
+      lit up and musical, Still OG and Alter Ego which looks like a hule in the
       wall but you have a feeling theres more than meets the eye, and Paper
       Planes which has a line around the block. You also see Concept barber shop
       which has its doors open and barbers barbering, Egghead Sando Cafe which
       strikes you as a nice breakfast and brunch spot, and the Elyse Restaurant
       which looks to be french cuisine.
     </p>
-    <ol>
-    <li><a href="">Go to Good Karma</a></li>
-    <li><a href="">Go to Subway</a></li>
-    <li><a href="">Go to CREAM</a></li>
-    <li><a href="">Go to Fifty-Five South</a></li>
-    <li><a href="">Go to Insomnia</a></li>
-    <li><a href="">Go to Nomkai</a></li>
-    <li><a href="">Go to Temple Bar and Lounge</a></li>
-    <li><a href="">Go to Fuze Lounge</a></li>
-    <li><a href="">Go to Still OG and Alter Ego</a></li>
-    <li><a href="">Go to Paper Planes</a></li>
-    <li><a href="">Go to Concept Barber Shop</a></li>
-    <li><a href="">Go to Egghead Sando Cafe</a></li>
-    <li><a href="">Go to Elyse Restaurant</a></li>
-    </ol>
+    <ul>
+      <li><a href="dead_end.html">Go to Good Karma</a></li>
+      <li><a href="dead_end.html">Go to Paper Planes</a></li>
```

Comment:
Can we make these more creative?

Maybe something like "Go inside the Good Karma and grab a beer"?